rfqmsg/messages.go

```diff
@@ -125,6 +125,13 @@ const (
 	V0 WireMsgDataVersion = 0
 )
 
+// Record returns a TLV record that can be used to encode/decode a
+// WireMsgDataVersion to/from a TLV stream.
+func (v *WireMsgDataVersion) Record() tlv.Record {
+	return tlv.MakeStaticRecord(0, v, 1, WireMsgDataVersionEncoder,
```

Comment:
nit: formatting of multi-line function call.

Reply:
Also, I know it's perhaps a bit too verbose, but I made it a habit to add the following comment wherever we use a 0 TLV type:
```
	// Note that we set the type here as zero, as when used with a
	// tlv.RecordT, the type param will be used as the type.
```